app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -0,0 +1,292 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
Please remove the license header.  It is already covered by the repo LICENSE files.  Besides, it's wrong anyway since we are dual licensed at this time